core/libdeno/api.cc

```diff
@@ -120,8 +120,8 @@ void deno_init() {
     // remove this to make it work asynchronously too. But that requires getting
     // PumpMessageLoop and RunMicrotasks setup correctly.
     // See https://github.com/denoland/deno/issues/2544
-    const char* argv[2] = {"", "--no-wasm-async-compilation"};
-    int argc = 2;
+    const char* argv[3] = {"", "--no-wasm-async-compilation", "--async-stack-traces"};
```

Comment:
I think `--async-stack-traces` is on by default. All it's specified to do is enrich `error.stack`: https://v8.dev/docs/stack-trace-api#async-stack-traces

Reply:
It doesn’t seem to have an effect - I was just trying it

Reply:
I think they have to be collected via other APIs 😢 see: https://github.com/nodejs/node/issues/11865#issuecomment-288450455